compiler/src/dotty/tools/dotc/reporting/Message.scala

```diff
@@ -146,7 +145,7 @@ object Message:
       }
 
       def addendum(cat: String, info: Type): String = info match {
-        case bounds @ TypeBounds(lo, hi) if !(bounds =:= TypeBounds.empty) =>
+        case bounds @ TypeBounds(lo, hi) if !(bounds =:= TypeBounds.empty) && !lo.isErroneous && !hi.isErroneous =>
```

Comment:
Does this work?
```suggestion
        case bounds @ TypeBounds(lo, hi) if !(bounds =:= TypeBounds.empty) && !bounds.isErroneous =>
```

Reply:
maybe, and maybe better when lo eq hi. I'll try it when I have sufficiently recovered from this PR.

Reply:
I should have just hit the commit suggestion button. My new year's resolution is to be bold in '24.